packages/configs/README.md

```diff
@@ -131,10 +131,10 @@ module.exports = {
   // add to `overrides` to prevent jest rules from applying to entire package
   overrides: [
     {
-      extends: ['amplify-ui/jest'],
+      extends: ['@aws-amplify/eslint-config-amplify-ui/jest'],
```

Comment:
I believe shorthand is still possible:
```suggestion
      extends: ['@aws-amplify/amplify-ui/jest'],
```

Can you try this and use shorthand if it works?

Reply:
I prefer not using the shorthand because it doesn't provide much benefits. If we didn't use the short-hand, it'd be easier to global search and find all of the places need to be changed.

Reply:
Happy to discuss, but I think the prefix `"eslint-config-"` is extraneous. Shorthand is a common syntax in eslint ecosystem as well and what we've been using, so prefer if we don't add additional changes to the PR.

Reply:
Though I still don't see the benefit of using the shorthand here, just updated the PR to unblock the release 👿